docs/api/verify-service-provider-api.swagger.yml

```diff
@@ -167,9 +175,9 @@ components:
           type: string
           enum:
             - AUTHENTICATION_FAILED
-          description: Returned when there was a failure to authenticate.
+          description: Returned when the `urn:oasis:names:tc:SAML:2.0:status:AuthnFailed` status code is included in the SAML response from the GOV.UK Verify Hub.
     NoAuthenticationScenario:
-      description: Returned when there wasn't an authentication
+      description: Returned when the user chose to cancel at some point in their journey.
```

Comment:
We should specify the SAML status code here.

Also, this won't probably won't happen due to a cancellation. It's more likely to happen due to a timeout.

Reply:
It's `urn:oasis:names:tc:SAML:2.0:status:NoAuthnContext` FYI

Reply:
```suggestion
      description: Returned when the user chose to cancel at some point in their journey. The SAML response for the GOV.UK Verify Hub contains the `urn:oasis:names:tc:SAML:2.0:status:NoAuthnContext` status code.
```